app/app.go

```diff
@@ -170,10 +150,9 @@ var (
 		vesting.AppModuleBasic{},
 		tokenmodule.AppModuleBasic{},
 		collectionmodule.AppModuleBasic{},
-		ibc.AppModuleBasic{},
-		transfer.AppModuleBasic{},
-		ica.AppModuleBasic{},
-		wasmplus.AppModuleBasic{},
+		rollup.AppModuleBasic{},
+		// ordamodule.AppModuleBasic{},
```

Comment:
Why comment out da module?

Reply:
This procedure is a direct migration of the `finschia-sdk` code. It's true that no `x/or/da` modules have been added. In conjunction with the points made subsequently, this part of the code will be reinstated.

Reply:
If you want to remove the comment, please import the latest commit.
https://github.com/Finschia/finschia-sdk/commit/25043656ed1a6a4947b2c5c3804d724bbcfc43ec